src/cmd/singularity/cli/pull.go

```diff
@@ -13,19 +13,25 @@ import (
 )
 
 var (
+	// PullLibraryURI holds the base URI to a Sylabs library API instance
 	PullLibraryURI string
 )
 
 func init() {
 	pullCmd.Flags().StringVar(&PullLibraryURI, "libraryuri", "http://localhost:5150", "")
+	pullCmd.Flags().BoolVarP(&Force, "force", "F", false, "overwrite an image file if it exists")
 	singularityCmd.AddCommand(pullCmd)
 
 }
 
 var pullCmd = &cobra.Command{
-	Use:  "pull myimage.sif library://user/collection/container:tag",
-	Args: cobra.ExactArgs(2),
+	Use:  "pull [options] [myimage.sif] library://user/collection/container[:tag]",
+	Args: cobra.RangeArgs(1, 2),
 	Run: func(cmd *cobra.Command, args []string) {
-		libexec.PullImage(args[0], args[1], PullLibraryURI)
+		if len(args) == 2 {
```

Comment:
Just a comment.  In general I think that we should push the runscripts elsewhere so that the CLI remains the CLI and they just call out to another function.  But there are several other instances of this that also need to be worked on.  I won't block b/c of it.  Just a comment.